backend/typescript/graphql/types/adminType.ts

```diff
@@ -0,0 +1,28 @@
+import { gql } from "apollo-server-express";
+
+const adminType = gql`
+  type NotificationDTO {
+    id: ID!
+    message: String!
+    createdAt: String!
+    residents: [NotificationResidentDTO!]
+  }
+
+  type NotificationResidentDTO {
+    notificationId: ID!
+    residentId: ID!
+    seen: Boolean!
+  }
+
+  extend type Query {
+    notificationById(id: ID!): NotificationDTO!
+    getAllNotifications: [NotificationDTO]
```

Comment:
I think this should be [NotificationDTO!] from my understanding since the notification list should contain valid notification and not null values.